cruise.umple/test/cruise/umple/compiler/024_multiAssocToSameClassWithMultiRoleName.ump

```diff
@@ -0,0 +1,7 @@
+class Person{
+}
+
+class Course{
+  * courseProf -- 1 Person prof;
+  * courseStudent -- 1..* Person students;
+}
```

Comment:
Your code doesn't pas the following example:
```
associationClass Friendship {
  * inviterFriend Profile invitee;
  * inviteeFriend Profile inviter;
  status;
}
class Profile {
    name;
}
```
maybe you wanted to have another PR for it?

Reply:
Yes, that will be another PR